src/Symfony/Component/Process/Tests/ProcessTest.php

```diff
@@ -1556,7 +1602,6 @@ public function testNotTerminableInputPipe()
 
     /**
      * @param string|array $commandline
-     * @param mixed        $input
```

Comment:
I didn't want to touch this but Fabbot code style tests asks to remove it. Bringing it up just for clarity.